docs/core/tutorials/using-on-macos.md

```diff
@@ -1,76 +1,55 @@
 ---
-title: Getting started with .NET Core on macOS
-description: Getting started with .NET Core on macOS, using Visual Studio Code
-keywords: .NET, .NET Core
+title: Getting started with .NET Core on macOS | Microsoft Docs
+description: This document provides the steps and workflow to create a .NET Core Solution using Visual Studio Code.
+keywords: .NET, .NET Core, Mac, macOS, Visual Studio Code
 author: bleroy
 ms.author: mairaw
-ms.date: 02/08/2017
+ms.date: 03/22/2017
 ms.topic: article
 ms.prod: .net-core
 ms.devlang: dotnet
 ms.assetid: 8ad82148-dac8-4b31-9128-b0e9610f4d9b
 ---
 
-# Getting started with .NET Core on macOS, using Visual Studio Code
+# Getting started with .NET Core on macOS
 
-This document provides a tour of the steps and workflow to create a .NET
-Core Solution using [Visual Studio Code](http://code.visualstudio.com).
-You'll learn how to create projects, create unit tests, use the debugging
-tools, and incorporate third-party libraries via [NuGet](http://nuget.org).
+This document provides the steps and workflow to create a .NET Core Solution using [Visual Studio Code](http://code.visualstudio.com). Learn how to create projects, create unit tests, use the debugging tools, and incorporate third-party libraries via [NuGet](https://www.nuget.org/).
 
-This article uses Visual Studio Code on Mac OS. Where there are differences,
-it points out the differences for the Windows platform.
+This article uses Visual Studio Code on Mac OS. Where there are differences, it points out the differences for the Windows platform.
```

Comment:
There's a repetition of Visual Studio Code here and in the previous paragraph -- it would be nice to eliminate it. I'm also not sure what differences this sentence has in view -- presumably, UI differences between VSCode on Windows and Mac, but it's not clear to me why they're very important.
I'd consolidate the two references to VSCode into a single alert box:
> [!NOTE]
> This article uses [Visual Studio Code](code.visualstudio.com) on MAC OS.